### ISSUE TITLE
Checkbox Group Is Missing a <legend> Element

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1753]
Location / URL: https://edac-vpat.instawp.co/wp-admin/admin.php?page=accessibility_checker_settings 

Screenshot: 

<img width="775" height="516" alt="Image" src="https://github.com/user-attachments/assets/f15b9b39-6fc5-4108-9bc8-7af255d073f0" />

WCAG S.C: 1.3.1 Info and Relationships (Level A)
Issue Description: The checkbox group is contained within a <fieldset>, but its group label ("Post Types To Be Checked") is implemented as a <label> associated with the first checkbox instead of a <legend>. As a result, the group label is not programmatically associated with the checkbox group, preventing assistive technologies from correctly conveying the relationship between the group label and its options.

**Comment by mjoslyn** [comment]
@pattonwebz - I can pick this up
